docs/reference/modules/coordination.asciidoc

```diff
@@ -0,0 +1,461 @@
+[[modules-cluster-coordination]]
+== Cluster coordination
+
+The cluster coordination module is responsible for electing a master node and
+managing changes to the cluster state.
+
+[float]
+=== Quorum-based decision making
+
+Electing a master node and changing the cluster state both work robustly by
+using multiple nodes, only considering each action to have succeeded on receipt
+of responses from a majority of the master-eligible nodes in the cluster. The
```

Comment:
This sentence sounds too complicated.

Reply:
I reworded a bit, but I'm still not 100% happy with it.

Reply:
I deleted words until I liked it.